conftest.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
```

Comment:
File in wrong place?

Reply:
Removed second, not needed conftest.py file.